lib/cnab/version.rb

```diff
@@ -1,3 +1,3 @@
 module Cnab
-  VERSION = "0.4.3"
+  VERSION = "0.4.4"
```

Comment:
Since you are adding a new feature this should increase the MINOR version instead of the patch one.
anyways, could you not do this in your PR and leave this to be done at the release process please?